M2/Macaulay2/packages/PackageCitations.m2

```diff
@@ -208,7 +214,7 @@ iCite String := S -> (
     if S === "M2" then return (
         concatenate (
             "@misc{M2,\n",
-            "  author = {Grayson, Daniel R. and Stillman, Michael E.},\n",
+            "  author = {Grayson, Daniel R. and Stillman, Michael E. and The Macaulay2 Authors},\n",
```

Comment:
This seems recursive to me, and I would almost prefer not changing it.
Also, it should probably be written as `{the {M}acaulay2 authors}`, but you should test that it appears correctly.

Reply:
Yeah, I'm not a huge fan of it either.  Specific person, specific person, very general group of people.  I just seem to remember maybe that's what we agreed on in M2internals?  :shrug: